DOCS-11819-mqtt3-connector-final-kt (#1611)
mulesoft/docs-connectors: mqtt 3/1.0.0/modules/ROOT/pages/mqtt3-connector-publish.adoc

= Publish Messages to an MQTT Broker with MQTT 3 Connector

> In these examples, you configure the Anypoint Connector for MQTT 3 (MQTT 3 Connector) *Publish* (`<mqtt3:publish>`) operation that enables you to publish a message to the desired topic (`topic`) and with a specific quality of service (`qos`).

Reviewer: ```suggestion
In these examples, you configure the Anypoint Connector for MQTT 3 (MQTT 3 Connector) *Publish* (`<mqtt3:publish>`) operation, which enables you to publish a message to the desired topic (`topic`) with a specific quality of service (`qos`).
```
